ui/helpers/utils/permission.js

```diff
@@ -185,7 +185,7 @@ const PERMISSION_DESCRIPTIONS = deepFreeze({
     permissionValue,
   ) => {
     const baseDescription = {
-      leftIcon: 'fas fa-info',
+      leftIcon: <Icon name={ICON_NAMES.INFO} />,
```

Comment:
Hmm not sure this would work? Here it's asking for a className and you're now passing it a component?

Reply:
Updated